ajax/libs/vue-functional-calendar/package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "vue-functional-calendar",
+  "title": "Vue.js Functional Calendar",
+  "description": "Lightweight, high-performance calendar component based on Vue.js",
+  "keywords": [
+    "vue",
+    "calendar",
+    "component",
+    "vue-component",
+    "functional",
+    "date picker",
+    "date range",
+    "range",
+    "multiple calendars",
+    "multiple",
+    "modal calendar",
+    "modal",
+    "inline",
+    "date",
+    "change month",
+    "change year",
+    "picker",
+    "day",
+    "month",
+    "year",
+    "time",
+    "add"
+  ],
+  "homepage": "https://github.com/ManukMinasyan/vue-functional-calendar",
+  "license": "MIT",
+  "author": "Manuk Minasyan <manuk.minasyan1@gmail.com> (https://github.com/ManukMinasyan)",
+  "filename": "FunctionalCalendar.umd.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "npmName": "vue-functional-calendar",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ManukMinasyan/vue-functional-calendar.git"
+  },
+  "version": "2.8.71"
```

```suggestion
  }
```
Remove `version` attr, we don't need it. Auto-update bot will add it.